src/google/protobuf/atomic_store.cc

```diff
@@ -0,0 +1,82 @@
+// --- start __atomic_store
+#if defined (__MVS__)
```

Comment:
I don't think we can accept this much special code for just z/OS. Does z/OS not support C++11 atomics?

Reply:
The compiler on zOS machine lacks this feature, we had to add it manually since the build requires that and we've faced link error.